### ISSUE TITLE
How does one actually compile from scratch?

### ISSUE THREAD

**Issue by evanberkowitz** [tomfaulkenberry/JASPbook#3]
Is there a makefile hidden somewhere?  In the `build` directory, there is a simple `lsj.tex` but when I run `pdflatex lsj.tex` on a clean repo `c03797c` I get

```
! Undefined control sequence.
l.4 \abx@aux@sortscheme
                       {nyt}
```

When I hit [enter] to just push past it I encounter all sorts of additional undefined control sequences.

**Comment by tomfaulkenberry** [comment]
No makefile...your `pdflatex` command should work.   There is probably some left-over code hanging around in the `.aux` file.  When this happens, deleting everything except the `.tex` usually does the trick.

I have cleaned up the build directory, so try pulling those changes and recompiling.  Let me know what happens!

Note - I've added some rudimentary build instructions to the `build` directory.  Thanks for bringing this up.
